.azure/vs2022.yml

```diff
@@ -1540,9 +1545,61 @@ stages:
           - publish: $(Build.StagingDirectory)
             artifact: devtools-$(arch)
 
+  - stage: swift_format
+    dependsOn: [toolchain, sdk, devtools]
+    jobs:
+      - job: build
+        strategy:
+          matrix:
+            'x64':
+              arch: amd64
+              platform: x64
+            # TODO(compnerd) enable AArch64 toolchain
+            # 'arm64':
+            #   arch: arm64
+            #   platform: aarch64
+        steps:
+          - task: DownloadPipelineArtifact@2
+            inputs:
+              buildType: current
+              artifactName: 'toolchain-amd64'
+          - task: DownloadPipelineArtifact@2
+            inputs:
+              buildType: current
+              artifactName: 'windows-sdk-$(arch)'
+          - task: DownloadPipelineArtifact@2
+            inputs:
+              buildType: 'current'
+              artifactName: 'devtools-amd64'
+          - checkout: self
+            fetchDepth: 1
+          - checkout: apple/swift-format
+            fetchDepth: 1
+          - task: PowerShell@2
+            inputs:
+              targetType: 'inline'
+              script: |
+                # Prepare input data for build.ps1
+                New-Item -ItemType Directory -ErrorAction Ignore $(Agent.BuildDirectory)/$(platform) | Out-Null
+                New-Item -ItemType Directory -ErrorAction Ignore $(Agent.BuildDirectory)/7/bin | Out-Null
```

Comment:
This also is a bit of a discomfort - the numbers have been renumbered in the past.  We shouldn't be relying on them to be consistent across builds.